matlab/make.m

```diff
@@ -1 +1 @@
-mex find_embedding.cpp -I../include CXXFLAGS="-std=c++11 -fPIC -O0" -lut -largeArrayDims -v
+mex find_embedding.cpp -I../include CXXFLAGS="-std=c++1y -fPIC -O3" -lut -largeArrayDims -v
```

Comment:
It's probably better to use `c++14` now (after the standard became official).

Reply:
My understanding is that Adam tried that, and it broke several compilers that we support.  I've deliberately avoided use of c++14 features for that reason.

Reply:
I don't know which compilers we're supporting, but there's [this](https://stackoverflow.com/questions/28825113/difference-between-gcc-compile-options-std-c1y-and-std-c14).

Reply:
Hmm.  Now I really don't know.  My suspicion is that Visual Studio was the source of problems and I might have broken it with the change to +1y

Reply:
in my personal opinion, both windows and matlab should be burned and stricken from history books but since I can't control what platforms our users are on...

Reply:
According to [this](https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=vs-2017), `/std` option was introduced only in MSVC 2017, and it supports c++14. I'm not sure if and/or how `-std=c++1y` works on M$ at all. Not my platform of choice either. :)

Reply:
I'm going to see if I can't test `std=c++14` on these platforms... I know Adam had some environments set up for that but this might take some digging

Reply:
:+1: